packages/components/pie-assistive-text/src/defs-react.ts

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import type React from 'react';
```

Comment:
gosh I'd be curious to see if this makes a wild different in bundle output

Reply:
I was curious about this too, so I tested it and it turns out it does not reduce the size, although I think that's because we already explicitly exclude `react` packages from the bundle.